caffe2/core/blob_test.cc

```diff
@@ -538,7 +538,13 @@ TEST(TensorTest, Tensor64BitDimension) {
   EXPECT_EQ(tensor.ndim(), 1);
   EXPECT_EQ(tensor.dim(0), large_number);
   EXPECT_EQ(tensor.size(), large_number);
-  EXPECT_TRUE(tensor.mutable_data<char>() != nullptr);
+  try {
+    EXPECT_TRUE(tensor.mutable_data<char>() != nullptr);
+  } catch (const EnforceNotMet& e) {
+    string msg = e.what();
+    msg = msg.substr(0, msg.find('\n'));
+    LOG(WARNING) << msg;
```

Comment:
You may want to make sure that the error message contains "aligned_malloc" - for all others, we should still not swallow the exception.

Reply:
Updated as well.